pkg/interface/src/views/components/StatusBar.tsx

```diff
@@ -36,7 +36,17 @@ const StatusBar = (props) => {
   const invites = [].concat(
     ...Object.values(inviteState).map(obj => Object.values(obj))
   );
-  const metaKey = window.navigator.platform.includes('Mac') ? '⌘' : 'Ctrl+';
+
+  let metaKey = 'Super+';
+  if (window.navigator.platform.includes('Mac')) {
+    metaKey = '⌘';
+  } else if (window.navigator.platform.includes('Win')) {
+    metaKey = '⊞ Win+';
+  }
+
+  const leapKey: string = useSettingsState(
+    useCallback(s => s.keyboard['leap'], ['leap'])
+    )?.replace('meta+', metaKey) || '';
```

Comment:
Most of this should be defined outside the scope of the component and then just pass the function to useSettingsState

Reply:
I never really understood how that worked with CalmEngine, can you give an example?

Reply:
```typescript
// outside component
  let metaKey = 'Super+';
  if (window.navigator.platform.includes('Mac')) {
    metaKey = '⌘';
  } else if (window.navigator.platform.includes('Win')) {
    metaKey = '⊞ Win+';
  }
const selKey = (s: SettingsState) => s.keyboard.leap.replace('meta+', metaKey);

// inside component

const leapKey = useSettingsState(selKey);
```